modules/weather.md

```diff
@@ -48,6 +48,8 @@ The following properties can be configured:
 | `showPeriod`                 | Show the period (am/pm) with 12 hour format <br><br> **Possible values:** `true` or `false` <br> **Default value:** `true`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 | `showPeriodUpper`            | Show the period (AM/PM) with 12 hour format as uppercase <br><br> **Possible values:** `true` or `false` <br> **Default value:** `false`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `showPrecipitationAmount` | Show the amount of rain/snow in the forecast <br><br>Not supported by all providers, and some providers does not support all `type`s. Supported providers are `envcanada`, `openmeteo`, `openweathermap`, `smhi`, `weatherbit` and `yr`.<br><br> **Possible values:** `true` or `false` <br> **Default value:** `false`                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
its not only in the forecast but also in the current weather

Reply:
also a small typo "does" -> "do".

both comment apply to the percentage section too